apps/angular/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -1,13 +1,42 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, ViewEncapsulation } from '@angular/core';
+import { CityStore } from '../../data-access/city.store';
+import { FakeHttpService } from '../../data-access/fake-http.service';
+import { CardType } from '../../model/card.model';
+import { City } from '../../model/city.model';
+import { CardComponent } from '../../ui/card/card.component';
 
 @Component({
   selector: 'app-city-card',
-  template: 'TODO City',
+  template: `
+    <app-card
+      [list]="cities"
+      [type]="cardType"
+      [imageSrc]="'assets/img/city.png'"
+      customClass="bg-light-blue"></app-card>
+  `,
+  encapsulation: ViewEncapsulation.None,
```

Comment:
FYI: it's dangerous to set `ViewEncapsulation.None`. In this case right here, the styling for `bg-light-blue` is set globally. So if you'd have the same css class in a different component, this styles would be applied.